serverless-heaven/serverless-webpack
# Add support for multiple runtimes in service

## What did you implement:

Added support for projects that have lambdas running on multiple runtimes. In our case, we needed some java lambdas to run in the same project as our typescript lambdas and the plugin was trying to bundle all runtimes.

## How did you implement it:

In validate.js, added a check for runtime.startswith('node') so that the plugin only tries to package node lambdas.

## How can we verify it:

Create a serverless project that includes node lambdas, and lambdas from other supported runtimes (java, python, etc). Make sure that package.invidually is true. Master branch will try to package the other runtimes with webpack and will cause errors when invoking the lambdas. This fork will package only node lambdas with the plugin

## Todos:

- [X] Write documentation (None needed)

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
